feat(Style guide): Added the image annotations page (#10888)
newrelic/docs-website: src/content/docs/style-guide/images/screenshots-images.mdx

          </td>

          <td>

>             Make sure your caption summarizes what the overall flow chart or diagram intends to communicate. See our [captions tips](/docs/style-guide/images/screenshots-images/#captions).

Reviewer: ```suggestion
            Make sure your caption summarizes what the overall flow chart or diagram intends to communicate. See our [captions tips](/docs/style-guide/images/screenshots-images#captions).
```

Author: Already done it.

    Keep in mind New Relic Security guidelines and avoid private information from New Relic or from customers. In addition:

    * Always crop or mask sensitive information.
    * Annotate images sparingly.

>     * Consider adding a caption. See our [tips](/docs/style-guide/images/screenshots-images/#captions) below.

Reviewer: ```suggestion
    * Consider adding a caption. See our [tips](/docs/style-guide/images/screenshots-images#captions) below.
```

Author: Already done it.

>     * Avoid having non-UI elements like the mouse pointer or the browser's scrollbar.

Reviewer: ```suggestion
    * Make sure to crop non-UI elements, such as your mouse pointer or browser tabs.
```

>     * If you need to blur part of the screenshot, use the gaussian blur. 

Reviewer: ```suggestion
    * If you need to blur part of the screenshot, use the blur tool in Snagit. 
```
Consider adding an image here^

Author: @a-sassman, I think that the gaussian blur was indicated by @austin-schaefer. Austin, can you confirm this? or maybe I've made it up, I can't remember now...

Reviewer 2: @nbaenam I'm seeing eight blur options in SnagIt. In the past we've used the first one. Could one of these be gaussian?

</CollapserGroup>

## Delete or rename an image file [#delete-rename-image]

> Because our translated mdx files in the i18n folder reference the same image folder that our English-language mdx files do, when image files are deleted or renamed, this can throw a build error because of the mismatch between the English files and the translated files. Here are some tips for deleting or renaming an image file: 

Reviewer: ```suggestion
Our i18n translated .mdx files need to match the corresponding English .mdx files. When you delete or rename image files in English, you might cause a build error because there's now a mismatch between the English and translated files. Here are some tips when deleting or renaming an image file: 
```

Author: Added the modification! Thanks!
